_redirects

```diff
@@ -1,3 +1,81 @@
+/docs/dbt-cloud/release-notes /docs/dbt-versions/dbt-cloud-release-notes 302
+/docs/dbt-cloud/dbt-cloud-enterprise/audit-log /docs/collaborate/manage-access/audit-log 302
+/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-setting-up-bigquery-oauth /docs/collaborate/manage-access/set-up-bigquery-oauth 302
+/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-snowflake-oauth /docs/collaborate/manage-access/set-up-snowflake-oauth 302
+/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-okta /docs/collaborate/manage-access/set-up-sso-okta 302
+/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-enterprise-sso-with-azure-active-directory /docs/collaborate/manage-access/set-up-sso-azure-active-directory 302
+/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-google-gsuite /docs/collaborate/manage-access/set-up-sso-google-gsuite 302
```

Comment:
"gsuite" is retired nomenclature.  Can we replace it with "workspace"?

Reply:
great flag! thanks!

Reply:
fixed the filename and redirect! thanks, @matthewshaver !